lib/myelements/add_objects.py

```diff
@@ -530,17 +540,21 @@ def convexPoly(
 
             Return: box2d.b2Body
         """
-        # NOTE: Box2D has a maximum poly vertex count, defined in Common/box2d.b2Settings.h (box2d.b2_maxPolygonVertices)
-        # We need to make sure, that we reach that by reducing the poly with increased tolerance
+        # NOTE: Box2D has a maximum poly vertex count,
+        # defined in Common/box2d.b2Settings.h (box2d.b2_maxPolygonVertices)
+        # We need to make sure, that we reach that by reducing the poly with
+        # increased tolerance
         # Reduce Polygon
         tolerance = 10  # 5
         v_new = vertices
         while len(v_new) > box2d.b2_maxPolygonVertices:
             tolerance += 1
             v_new = tools_poly.reduce_poly(vertices, tolerance)
 
-        print("convexPoly: Polygon reduced from %i to %i vertices | tolerance: %i" % (
-            len(vertices), len(v_new), tolerance))
+        msg = ("convexPoly: Polygon reduced from %i to %i vertices | "
+               "tolerance: %i")
+        print(msg % (len(vertices), len(v_new), tolerance))
+
```

Comment:
I'm guessing this was a debug need, you can use git blame and look at the particular commit that introduced it and see if there's any explanation and if there isn't then you can use logging.debug for this if it's used in other places or just remove it entirely.